snoop/data/management/commands/filestats.py

```diff
@@ -0,0 +1,128 @@
+    'application/x-zip',
+    'application/x-rar',
+    'application/rar',
+    'application/x-gzip',
+    'application/gzip',
+    'application/x-bzip2',
+    'application/x-tar',
+    'application/x-hoover-pst', 
+    'application/mbox',
+    'application/pdf',
+
+    'text/plain',
+    'text/html',
+    'text/rtf',
+
+    'application/pdf',
+
+    'application/msword',
+    'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+    'application/vnd.openxmlformats-officedocument.wordprocessingml.template',
+    'application/vnd.ms-word.document.macroEnabled.12',
+    'application/vnd.ms-word.template.macroEnabled.12',
+    'application/vnd.oasis.opendocument.text',
+    'application/vnd.oasis.opendocument.text-template',
+    'application/rtf',
+
+    'application/vnd.ms-excel',
+    'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+    'application/vnd.openxmlformats-officedocument.spreadsheetml.template',
+    'application/vnd.ms-excel.sheet.macroEnabled.12',
+    'application/vnd.ms-excel.template.macroEnabled.12',
+    'application/vnd.ms-excel.addin.macroEnabled.12',
+    'application/vnd.ms-excel.sheet.binary.macroEnabled.12',
+    'application/vnd.oasis.opendocument.spreadsheet-template',
+    'application/vnd.oasis.opendocument.spreadsheet',
+
+    'application/vnd.openxmlformats-officedocument.presentationml.presentation',
+    'application/vnd.openxmlformats-officedocument.presentationml.template',
+    'application/vnd.openxmlformats-officedocument.presentationml.slideshow',
+    'application/vnd.ms-powerpoint',
+    'application/vnd.ms-powerpoint.addin.macroEnabled.12',
+    'application/vnd.ms-powerpoint.presentation.macroEnabled.12',
+    'application/vnd.ms-powerpoint.template.macroEnabled.12',
+    'application/vnd.ms-powerpoint.slideshow.macroEnabled.12',
+    'application/vnd.oasis.opendocument.presentation',
+    'application/vnd.oasis.opendocument.presentation-template',
+
+    'application/xhtml+xml',
+    'application/xml',
+    'text/xml',
+    'image/tiff',
+    'image/jpg',
+    'image/webp',
+    'image/heic',
+
+    'application/vnd.ms-outlook',
+    'application/vnd.ms-office',
+    'application/CDFV2',
+    'message/rfc822',
+    'message/x-emlx',
+}
+
+def truncateSize(size):
+    exponent = len(str(size))-1
+    divisor = 10**exponent
+    return int((size/divisor))*divisor
+
+class Command(BaseCommand):
+    help = "Display filetype stats."
+
+    def add_arguments(self, parser):
+        parser.add_argument('collection', type=str)
+
+    def handle(self, collection, **options):
+
+        col = collections.ALL[collection]
+        
+        with col.set_current():
+            querysetMime = models.Blob.objects.all().values('mime_type').annotate(total=Count('mime_type')).annotate(size=Sum('size')).order_by('-size')
+            print()
+            print('---- Top 100 Mime-Types by size ----')
+            print()
+            for mtype in querysetMime:
```

Comment:
make some top-level functions in this file called `get_top_mime_types()` and `get_top_extensions()` that return python dicts with the data you want to print() in this command here. This has the added benefit we can later just move your functions somewhere else in the code base and display this info in the front-end too.